incubator/zookeeper/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: zookeeper
 home: https://zookeeper.apache.org/
-version: 2.1.4
+version: 2.1.5
```

```suggestion
version: 2.2.0
```
this PR add a new feature for the chart, not a bug fix